core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
> For instance, a bridge section over a river has 'height' attribute set to 60m.

Does it have a height or maxheight of 60m? Height would be correctly mapped. Maxheight would be wrong. If it is maxheight, we could report an error to OSM.